br.app.pw3270.terminal.yml

```diff
@@ -0,0 +1,43 @@
+app-id: br.app.pw3270.terminal
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: pw3270
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/man"
+  - "/share/doc"
+  - "/share/gtk-doc"
+  - "/share/man"
+  - "/share/pkgconfig"
+  - "*.la"
+  - "*.a"
+modules:
+  - name: lib3270
+    buildsystem: autotools
+    config-opts: 
+      - --with-product-id="br.app.pw3270"
+    sources:
+      - type: git
+        tag: "5.4"
```

Comment:
generate a tarball of source code with sha256, it would be better to use tags/releases from github itself

Reply:
I prefer to use the automated way with tags os branches. It's far more easy to maintain than generating a tarball manually on every update. And, it's the method used for the other packaging tool I'm using.

Reply:
git tag is perfectly fine.